packages/stockflux-components/package.json

```diff
@@ -1,32 +1,33 @@
 {
-  "name": "stockflux-components",
-  "version": "13.0.0",
-  "private": true,
-  "main": "./dist/index.js",
-  "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "babel-loader": "^8.0.6",
-    "css-loader": "^2.1.1",
-    "date-fns": "^1.30.1",
-    "openfin-fdc3": "^0.1.1",
-    "stockflux-core": "13.0.0",
-    "style-loader": "^0.23.1",
-    "url-loader": "^1.1.2",
-    "webpack": "^4.39.3",
-    "webpack-cli": "^3.3.8"
-  },
-  "peerDependencies": {
-    "react": "^16.8.6",
-    "openfin-react-hooks": "^1.0.1"
-  },
-  "scripts": {
-    "build": "webpack"
-  },
-  "dependencies": {
-    "classnames": "^2.2.6",
-    "react-icons": "^3.7.0",
-    "react-svg-loader": "^3.0.3"
-  }
+    "name": "stockflux-components",
+    "version": "13.0.0",
+    "private": true,
+    "main": "./dist/index.js",
+    "devDependencies": {
+        "@babel/cli": "^7.5.5",
+        "@babel/core": "^7.5.5",
+        "@babel/preset-react": "^7.0.0",
+        "babel-loader": "^8.0.6",
+        "css-loader": "^2.1.1",
+        "date-fns": "^1.30.1",
+        "openfin-fdc3": "^0.1.1",
+        "stockflux-core": "13.0.0",
+        "style-loader": "^0.23.1",
+        "url-loader": "^1.1.2",
+        "webpack": "^4.39.3",
+        "webpack-cli": "^3.3.8"
+    },
+    "peerDependencies": {
+        "react": "^16.8.6",
+        "openfin-react-hooks": "^1.3.0"
+    },
+    "scripts": {
+        "build": "webpack"
+    },
+    "dependencies": {
+        "classnames": "^2.2.6",
+        "openfin-react-hooks": "^1.3.0",
```

Comment:
Has the spacing changed on the file? It looks as though the `openfin-react-hooks` are the only lines that have changed?

Reply:
Yep, thanks for noticing this. Should be fixed now.